### ISSUE TITLE
Pinia persistedstate multiple app?

### ISSUE THREAD

**Issue by fathan** [prazdevs/pinia-plugin-persistedstate#62]
can pinia persistedstate save different storage from the two vue applications? by using a different persist key? Thank you

**Comment by prazdevs** [comment]
Yeah ofc, just make sure to use different keys or storage to not override each other!

**Comment by fathan** [comment]
@prazdevs set prefix on $store.id is imposibble?

**Comment by prazdevs** [comment]
by default, the storage key is the store's id
